data/txHandler.go

```diff
@@ -107,6 +117,22 @@ func MakeTxHandler(txPool *pools.TransactionPool, ledger *Ledger, net network.Go
 		postVerificationQueue: make(chan *txBacklogMsg, txBacklogSize),
 		net:                   net,
 	}
+	handler.ctx, handler.ctxCancel = context.WithCancel(context.Background())
+	if cfg.EnableTxBacklogRateLimiting {
+		congestionManager := util.NewREDCongestionManager(
+			(time.Second * time.Duration(cfg.TxBacklogServiceRateWindowSeconds)),
+			100*cfg.TxBacklogServiceRateWindowSeconds, // Service Rates update 1/s @ 100 requests per second to the congestion manager
```

Comment:
This number feels too small. 

The usage of the "window size" was a bit confusing. 
I double checked and the congestion thread is looping and draining from the 3 queues using that size regardless of the value of `d`. 

I would not make the two things related in the init size since they have no relation.

In practice that value is related to the tx arrival rate and backlog service rate, since those are the instances where this queues see get an item added, and the rate at which we consume from those queues.

This service is much much faster than the tx backlog service, but we still need enough room for the arrivals.

I would make this the same size as the resource we are protecting.